python/paddle/distributed/fleet/meta_optimizers/recompute_optimizer.py

```diff
@@ -18,24 +18,31 @@
 class RecomputeOptimizer(MetaOptimizerBase):
     def __init__(self, optimizer):
         super(RecomputeOptimizer, self).__init__(optimizer)
-        #self.inner_opt = RO(optimizer)
         self.inner_opt = optimizer
-        self.wrapped_opt = RO(optimizer)
+        self.wrapped_opt = None
         # we do not allow meta optimizer to be inner optimizer currently
         self.meta_optimizers_white_list = [
             "LarsOptimizer",
             "LambOptimizer",
             "GradientMergeOptimizer",
```

Comment:
help remove this from the `meta_optimizers_white_list` of RecomputeOptimizer.

Reply:
removed